sources/packages/web/src/views/student/NoticeOfAssessment.vue

```diff
@@ -140,11 +143,11 @@ export default defineComponent({
             break;
           case ECertFailedValidation.MSFAACanceled:
           case ECertFailedValidation.MSFAANotSigned:
-            warningText.value.push(
-              `Your MSFAA is not valid. 
-              Please complete your MSFAA with the National Student Loans Centre to move forward with your application. 
-              Please note, there is a one day delay between signing your MSFAA and being able to accept your assessment.`,
-            );
+            if (!warningText.value.includes(MSFAAWarning)) {
```

Comment:
Can it have a "Canceled" and "Not signed" eCert validation for the same assessment?

Reply:
That is why I added this actually - the test case I was working with this time had that.

Reply:
I'm just wondering if that is a real possible scenario for production.

Reply:
I'm not sure if it would be realistic, but I don't think it hurts anything having a case there for it. Do you need me to remove it?

Reply:
ok. That's fine to keep it. Thanks.